src/modules/Hosts/HostsUILib/HostsMainPage.xaml

```diff
@@ -412,23 +412,23 @@
                 <ListView.ItemTemplate>
                     <DataTemplate x:DataType="models:Entry">
                         <Grid
+                            Margin="0"
                             AutomationProperties.Name="{x:Bind Address, Mode=OneWay}"
                             Background="Transparent"
                             ColumnSpacing="8">
                             <Grid.ColumnDefinitions>
-                                <ColumnDefinition Width="256" />
+                                <ColumnDefinition Width="*" MinWidth="150" />
                                 <!--  Address  -->
-                                <ColumnDefinition Width="*" />
+                                <ColumnDefinition Width="*" MinWidth="120" />
                                 <!--  Comment  -->
-                                <ColumnDefinition Width="Auto" />
+                                <ColumnDefinition Width="20" />
                                 <!--  Status  -->
-                                <ColumnDefinition Width="Auto" />
+                                <ColumnDefinition Width="20" />
                                 <!--  Duplicate  -->
-                                <ColumnDefinition Width="Auto" />
+                                <ColumnDefinition Width="40" />
                                 <!--  ToggleSwitch  -->
-                                <ColumnDefinition Width="Auto" />
+                                <ColumnDefinition Width="54" />
```

Comment:
Can we stick with auto on these columns width?

Reply:
Yep.

That's what I asked you. Are you okay when the ping and duplicate icons are centered instead of being vertical aligned? (See screenshot in PR description.)

Reply:
Changed the buttons back to auto. But I kept the fixed values for the icon to have the vertical alignment (See PR description.)